pkgs/top-level/all-packages.nix

```diff
@@ -3247,7 +3247,7 @@ with pkgs;
   keystone = callPackage ../development/libraries/keystone { };
 
   casync = callPackage ../applications/networking/sync/casync {
-    sphinx = python3Packages.sphinx;
+    sphinx = buildPackages.python3Packages.sphinx;
```

Comment:
sphinx is used in derivation as native build input. To avoid following issue, the recommended way is to use buildPackages:
https://github.com/NixOS/nixpkgs/issues/49526
I did not build or test this packages, but can you explain why you had to change this line?

Reply:
The splicing magic fails when parameters are explicitly passed to callPackage, thus it's required to prefix buildPackages to get the right sphinx.

Reply:
You are absolutely right. When I first read the diff, I thought you removed buildPackages. I missinterpreted the diff, for some reason.